students/ghassan/lesson04/trig.py

```diff
@@ -2,16 +2,28 @@
 
 from pprint import pprint
 
-with open('data.txt', 'r') as datafh:
-    data = datafh.read().split()
+''' with open('data.txt', 'r') as datafh:
+    data = datafh.read().split() '''
 
+data = "I wish I may I wish I might".split()
 z = {}
 
 
 for item in range(0, len(data)-2):
     z.setdefault('{} {}'.format(data[item], data[item+1]), [])
-    z['{} {}'.format(data[item], data[item+1])].append('{}'.format(data[item+2]))
+    z['{} {}'.format(
+        data[item], data[item+1])].append('{}'.format(data[item+2]))
 
-pprint(z)
+key = 'I may'
 
+new_file = [x for x in key.split()]
```

Comment:
`key.split()` will return the list for you already, so this can be just `new_file = key.split()`